openstack/designate/templates/seeds.yaml

```diff
@@ -38,8 +38,9 @@ spec:
   - name: dns_webmaster
   - name: dns_admin
   - name: dns_viewer
-  - name: cloud_identity_viewer
-  - name: cloud_resource_viewer
+  - name: role_viewer
```

Comment:
you don't need to add these 3 roles here, they are already introduced in other seeds - just make sure to have those covered under `spec:requires` above